spec/models/responsibility.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Responsibility, :type => :model do
+
+  describe '#by_church' do
+
+    context 'returns an array with the responsibilities of church' do
+      let!(:church) { create(:church)}
+      let!(:member) { create(:member, church: church)}
+      let!(:responsibility) { create(:responsibility)}
+      let!(:responsibilities) {[responsibility]}
+      let!(:ministry) { create(:ministry, church: church, responsibilities: responsibilities)}
+      let!(:charge) { create(:charge, responsibility:responsibility, ministry: ministry )}
+      let!(:charge_member) { create(:charge_member, charge:charge, member: member )}
```

Comment:
charge_member se necesita ya que está relacionado con charge, si no lo agrego me da error.

     Failure/Error: expect(Responsibility.by_church(church)).not_to be_empty
       expected `#<ActiveRecord::Relation []>.empty?` to return false, got true
     # ./spec/models/responsibility.rb:14:in `block (4 levels) in <top (required)>'